### PR TITLE
fix(project): fix/upgrade-to-vue-3-1-1-and-with-bug-fix

### PR DESCRIPTION
**Pull request by jw-foss**
- Upgrade Vue to 3.1.1 and fix the bug alone with the upgrade

Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by zouhangwithsweet — APPROVED** [review]

File: packages/time-picker/src/time-picker.ts [hunk 1]
```diff
@@ -32,15 +32,22 @@ export default defineComponent({
...
-      format: DEFAULT_FORMATS_TIME,
-      ...props, // allow format to be overwrite
-      type,
-      ref: commonPicker,
-      'onUpdate:modelValue': value => ctx.emit('update:modelValue', value),
-    },
-    {
-      default: scopedProps => h(panel, scopedProps),
-    })
+    console.log(props)
```
Comment (purepear) [inline packages/time-picker/src/time-picker.ts:35]: :)

File: packages/time-picker/src/time-picker-com/panel-time-range.vue [hunk 2]
```diff
@@ -276,6 +276,7 @@ export default defineComponent({
     }
 
     const formatToString = value => {
+      console.log(value, props.format)
```
Comment (wenfangdu) [inline packages/time-picker/src/time-picker-com/panel-time-range.vue:279]: is this necessary?

File: packages/time-picker/src/time-picker.ts (hunk 1 above)
Comment (JeremyWuuuuu) [reply packages/time-picker/src/time-picker.ts:35]: Nope. This was my bad. I will add an plugin to remove `console.log` statement to make sure this won't happen again

File: packages/time-picker/src/time-picker-com/panel-time-range.vue (hunk 2 above)
Comment (JeremyWuuuuu) [reply packages/time-picker/src/time-picker-com/panel-time-range.vue:279]: Nope. This was my bad. I will add an plugin to remove `console.log` statement to make sure this won't happen again

File: packages/time-picker/src/time-picker.ts (hunk 1 above)
Comment (JeremyWuuuuu) [reply packages/time-picker/src/time-picker.ts:35]: Just got it fixed. Upgrade to beta48 then you won't see those annoying logs.

File: packages/time-picker/src/time-picker-com/panel-time-range.vue (hunk 2 above)
Comment (JeremyWuuuuu) [reply packages/time-picker/src/time-picker-com/panel-time-range.vue:279]: I just fixed it. Upgrade your local version then it should be resolved.

**Pull request merged**